perl/cpan/3rdparty/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("//perl/cpan:perl_cpan_compiler.bzl", "perl_cpan_compiler")
+
+exports_files([
+    "cpanfile",
+    "cpanfile.snapshot",
+])
```

Comment:
Are these necessary? Aren't https://github.com/bazel-contrib/rules_perl/blob/85c5f5520da92662e71c073e164076e52b861cba/perl/cpan/3rdparty/BUILD.bazel#L10-L11 enough to bring them in?

Reply:
Eh yagni 🤷